repositories/createPost.ts

```diff
@@ -1,11 +1,44 @@
 import * as firebase from "firebase";
-import GameSituation from "../models/GameSituation";
+import { GameType, GameStreetAction } from "../models/GameSituation";
+import PlayingCard from "../models/PlayingCard";
 import { PostTitle, PostBody, PostId } from "../models/Post";
 
 export interface PostData {
   title: PostTitle;
   body: PostBody;
-  gameSituation: GameSituation;
+  gameSituation: PostingGameSituation;
+}
```

Comment:
`PostData` itself sounds a little bit tricky though, `PostingGameSituation` should be `PostDataGameSituation`, probably?